lib/immobilienscout/version.rb

```diff
@@ -0,0 +1,3 @@
+module Immobilienscout
+  VERSION = '0.0.2'
+end
```

Comment:
Version bump should usually be a separate commit on `master`, which can then be tagged for release.

For example: https://github.com/plataformatec/devise/commit/2e5b5fcd705b06c518ab0156b96badb91c4cb6ea

Reply:
Yes but this is not a bump, is the actual version

Reply:
Ah right makes sense!